maya/core.py

```diff
@@ -675,7 +675,7 @@ def now():
     return MayaDT(epoch=epoch)
 
 
-def when(string, timezone='UTC', prefer_past=False):
+def when(string, timezone='UTC', prefer_dates_from='current_period'):
```

Comment:
Can you document the possible values of `prefer_dates_from` in the docstring of `maya.when`?
Maybe also reference the dateparser documentation.

Reply:
oops, completely forgot the docstring! Should be updated now.